internal/venice-common/src/main/java/com/linkedin/venice/utils/Utils.java

```diff
@@ -461,15 +461,14 @@ public static boolean verifyTransition(ExecutionStatus newStatus, ExecutionStatu
   }
 
   public static List<String> parseCommaSeparatedStringToList(String rawString) {
-    String[] strArray = rawString.split(",\\s*");
-    if (strArray.length < 1) {
-      throw new VeniceException("Invalid input: " + rawString);
+    if (StringUtils.isEmpty(rawString)) {
+      return Collections.emptyList();
     }
-    return Arrays.asList(strArray);
+    return Arrays.asList(rawString.split(",\\s*"));
```

Comment:
When checking the uses of this function, I realized that it is no longer used. So, I've removed it now. I've replaced it's uses with `parseCommaSeparatedStringToSet`.

However, the comment is still valid as there could be a change in behavior. So, I inspected the uses of this function and have now verified that the "emptiness" of the Set is handled gracefully.

There was another change in behavior with this change for A/A configs. Previously, users would set `active.active.replication.enabled.on.controllers`, and if this is set, then we looked for AA configs like `active.active.real.time.source.fabric.list` and threw an exception. Now, with this change, since `active.active.replication.enabled.on.controllers` is no longer a valid config, I've made the default behavior of non-specified `active.active.real.time.source.fabric.list` to use the value of `native.replication.fabric.allowlist` - and this was indirectly a required config previously since in multi-region mode, NR is the only supported mode now.

Reply:
Thanks for checking.